src/components/Invoice/InvoiceForm.tsx

```diff
@@ -56,6 +59,7 @@ export default function InvoiceForm({ defaultValues }: InvoiceProps) {
                         {...field}
                         value={parseDate(field.value)}
                         slotProps={{ textField: { error: !!errors?.createdAt } }}
+                        onChange={(date) => field.onChange(format(date as Date, 'yyyy-MM-dd'))}
```

Comment:
this is a bit dangerous. If someone removes the date completely date here is null, so `format` will fail.

Reply:
changed